shared/chat/conversation/messages/attachment/image/index.js

```diff
@@ -255,21 +278,14 @@ const styles = Styles.styleSheetCreate({
     color: Styles.globalColors.black_40,
     marginRight: Styles.globalMargins.tiny,
   },
-  spinner: Styles.platformStyles({
-    isElectron: {
-      ...Styles.globalStyles.flexBoxColumn,
-      alignItems: 'center',
-    },
-    isMobile: {
-      ...Styles.globalStyles.flexBoxCenter,
-      alignItems: 'center',
-      flex: 1,
-      margin: 'auto',
-    },
-  }),
   title: Styles.platformStyles({
     isElectron: {
       wordBreak: 'break-word',
+      padding: 5,
+      display: 'block',
+    },
+    isMobile: {
+      padding: 5,
     },
```

Comment:
You could throw the padding into the `common` object if they're going to be the same. It could probably stand a few more pixels, too. Maybe `Styles.globalMargins.tiny`?